internal/io/seekable.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the LGPLv3, see COPYING and COPYING.LESSER file for details.
+
+package io
```

Comment:
Could we please not name this the same as the stdlib io package?

Reply:
done